kompose.spec

```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
```

Comment:
yeah but that is an answer to the question `How do I know if a variable is set in Bash?`. We don't care if the variable is set or not, just that it is not empty :). Good to know how to do that, though.